site/src/components/badge.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Badge
+description: The Badge component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fdesign%2FuVA2amRR71yJZ0GS6RI6zL%2F%25F0%259F%258C%259E-Atlas-Design-Library%3Fm%3Dauto%26node-id%3D3381-10722%26t%3DZhpJOvxIFDBwfkR7-1
+classType: Component
+classPrefixes:
+  - badge
+---
+
+# Badge
+
+Badges are informative, non-interactive components that add extra context around the assoiciated component.
+
+## Usage
+
+Badges use short text, color, and an optional icon for quick recognition.
+
+```html
+<span class="badge">Badge</span>
+```
+
+### Sizes
+
+The following classes are available for resizing: `badge-sm`, `badge-lg`.
+
+```html
+<span class="badge badge-sm">Badge</span>
+<span class="badge">Badge</span>
+<span class="badge badge-lg">Badge</span>
+```
+
+### Semantic colors & types
+
+By default, badges are secondary badges.
```

Comment:
Perhaps explain what secondary badges mean?

Reply:
I followed how it was written in [buttons](https://design.learn.microsoft.com/components/button.html):

![image](https://github.com/user-attachments/assets/7c8747dd-5015-4f14-9ab7-3c9ffd6f2e93)

But I also think having a separate "Semantic colors" page might be a good idea, cause there are several components already using them and I'd love to avoid writing the same thing everywhere.

Reply:
Sounds good. We can add a link to the new section later.